packages/service-library/src/servicelib/rabbitmq/_rpc_router.py

```diff
@@ -64,15 +64,17 @@ async def _wrapper(*args, **kwargs):
                         ):
                             raise
 
-                        _logger.exception("Unhandled exception:")
+                        _logger.exception(
+                            "Unhandled exception: %s", exc  # noqa: TRY401
```

Comment:
log.exception already contains the exception information.
see [https://docs.python.org/3/library/logging.html#logging.Logger.exception](https://docs.python.org/3/library/logging.html#logging.Logger.exception)

Reply:
I will revert this one. I got confused by the one below. It is not necessary to add any changes here.